default.yaml

```diff
@@ -167,6 +167,13 @@ onChainWallet:
   scanDepthOutgoing: 2
   scanDepthChannelUpdate: 8
 
+swap:
+  minOutboundLiquidityBalance: 30000000
```

Comment:
if there is a single parameter, shouldn't it be `minOnChainHotWalletBalance` ?

Reply:
yeah, spoke to Justin on Friday to clarify what he meant by that. I have a lot of fixes coming on Monday.

Reply:
fixed with https://github.com/GaloyMoney/galoy/pull/1396/commits/8c464b23670f31ff6d4866e93a16276eaadd1b63#diff-94d4d5bb5c048793536625edb8adffb17e89b5f3821cec6d1c7c337578cda8e4R171

we also check to make to make sure we have enough outbound `hasEnoughOutboundLiquidity` src/domain/swap/swap-out-checker.ts

```typescript
const hasEnoughOutboundLiquidity = currentOutboundLiquidityBalance > swapOutAmount
```